build.sbt

```diff
@@ -39,6 +39,11 @@ enablePlugins(ScalaUnidocPlugin)
 addCommandAlias("benchAll", benchAll.command)
 addCommandAlias("benchLSP", benchLSP.command)
 addCommandAlias("benchQuick", benchQuick.command)
+commands += Command.command("releaseSemanticdb")(s =>
+  "semanticdbSharedJVM/publishSigned" :: "semanticdbScalacPlugin/publishSigned" ::
```

Comment:
this begs for `(a :: b :: ...).map(_ + "/publishSigned") ++ s`, or something like that

Reply:
done!